### ISSUE TITLE
executed lines not marked as covered

### ISSUE THREAD

**Issue by rwwagner90** [sglanzer/ember-cli-blanket#115]
It would appear that `export` is the only thing not covered in my code. Shouldn't `export` be ignored or something? What's going on here?

![screen shot 2016-03-01 at 5 41 43 pm](https://cloud.githubusercontent.com/assets/2640861/13444564/0616129e-dfd5-11e5-9bf4-41d9113aa61f.png)

**Comment by jschilli** [comment]
is that a screenshot from the browser? or external report?

we know that the exports are being transpiled

what version of ember? ember-cli-blanket?

**Comment by rwwagner90** [comment]
@jschilli that is a screenshot from Code Climate. 

I'm on Ember 2.3.1 and ember-cli-blanket 0.8.0

Is it supposed to run coverage on the transpiled code and not the original code?

My options look like this, I might have messed up:

```
var options = {
  modulePrefix: 'ember-math-helpers',
  filter: '//.*ember-math-helpers/.*/',
  antifilter: '//.*(tests|template).*/',
  loaderExclusions: [],
  enableCoverage: true,
  cliOptions: {
    reporters: ['lcov'],
    autostart: true,
    lcovOptions: {
      outputFile: 'lcov.dat',
      renamer: function (moduleName) {
        var expression = /^ember-math-helpers/;
        return moduleName.replace(expression, 'addon') + '.js';
      }
    }
  }
};
```

**Comment by rwwagner90** [comment]
Okay, I looked at localhost:4200/tests, and it is transpiled. Now the uncovered part is the actual inside of the function. I'm not sure how it's not covered though. My test correctly does the addition, and it passes, so I am not sure why it is listed as not covered.

```js
test('addition works', function(assert) {
  let result = add([20, 10]);
  assert.equal(result, 30);
});
```

<img width="1126" alt="screen shot 2016-03-01 at 9 21 48 pm" src="https://cloud.githubusercontent.com/assets/2640861/13448789/cac46a8c-dff3-11e5-8bc0-9ce2076e11e4.png">

**Comment by rwwagner90** [comment]
@jschilli looks like this may be related to https://github.com/sglanzer/ember-cli-blanket/issues/105

This code is clearly exectued, but not marked as covered. I saw the fix there was to downgrade loader.js. Is there a way to fix this or update the library so we don't have to downgrade?

**Comment by rwwagner90** [comment]
This should be fixed with 0.9.1
